pkgs/build-support/xen/default.nix

```diff
@@ -103,11 +107,22 @@ let
     branch
     version
     latest
+    genericPatchList
     pkg
     ;
 
   # Mark versions older than minSupportedVersion as EOL.
-  minSupportedVersion = "4.17";
+  minSupportedVersion = "4.16";
```

Comment:
```suggestion
  minSupportedVersion = "4.18";
```

Reply:
I'd rather keep it at the upstream value. My reasoning is that people maintaining their own downstream Xen derivations might want to use the generic builder to build a 4.16 or 4.17 Xen, and since the updateScript that depended on grepping the value no longer exists, we have no reason to restrict the builder arbitrarily.

Reply:
Oh, I thought it was a typo during removal of 4.17